odl/contrib/datasets/ct/mayo.py

```diff
@@ -0,0 +1,270 @@
+# Copyright 2014-2017 The ODL contributors
+#
+# This file is part of ODL.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at https://mozilla.org/MPL/2.0/.
+
+"""Tomographic datasets from Mayo Clinic.
+
+In addition to the standard ODL requirements, this library also requires:
+
+    - tqdm
+    - dicom
+    - A copy of the Mayo dataset, see
+    https://www.aapm.org/GrandChallenge/LowDoseCT/#registration
```

Comment:
This is a good list -- can you put it in the readme as well?

Reply:
Both are pip installable, I'd say we just skip them for now since that would clog the readme